modules/integration/tests-integration/tests-backend/src/test/java/org/wso2/identity/integration/test/rest/api/challenge/questions/ServerChallengeTestCase.java

```diff
@@ -0,0 +1,346 @@
+/*
+ *Copyright (c) 2005-2010, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```

Comment:
check the license header. Seems like an outdated one

Reply:
Modified